spokestack/vad/webrtc.py

```diff
@@ -0,0 +1,74 @@
+"""
+This module contains the webrtc component for
+voice activity detection (vad)
+"""
+
+import webrtcvad  # type: ignore
+
+from spokestack.context import SpeechContext
+
+
+MODE = {"quality": 0, "low_bitrate": 1, "aggresive": 2, "very_aggresive": 3}
+
+
+class VoiceActivityDetector:
+    """ This class detects the presence of voice in a frame of audio.
```

Comment:
it would be good to generate the documentation for this module, then read it and see if it should be fleshed out more - think about whether this would be useful documentation for a newcomer to the library

Reply:
can I do this with the sphinx PR?

Reply:
I don't see why not